app/src/main/AndroidManifest.xml

```diff
@@ -14,11 +14,11 @@
 
 
         <activity
-            android:name=".BuscarActivity"
+            android:name=".SearchActivity"
             android:exported="false"
             android:theme="@style/buscar_activity"/>
         <activity
-            android:name=".ActivityRegistro"
+            android:name=".ActivityRegister"
```

Comment:
you have SearchActivity and ActivityRegister, please standardize the naming by having `Activity` as either prefix or suffix but not both.

Reply:
Thanks for the advice